docs/content/docs/test/Runner.md

```diff
@@ -5,14 +5,34 @@ type: docs
 
```

Comment:
I've created a single issue (MBS-8122) for full documentation.
In this PR I want just to translate. This page will be an entrypoint for the product in an article.